nucliadb/nucliadb/search/search/query.py

```diff
@@ -51,9 +51,47 @@
 from nucliadb_utils import const
 from nucliadb_utils.utilities import has_feature
 
+from .exceptions import InvalidQueryError
+
 ENTITY_FILTER_PREFIX = "/e/"
 LABEL_FILTER_PREFIX = "/l/"
 
+LABEL_QUERY_ALIASES = {
+    # aliases to make querying labels easier
+    "tags": ["t"],
+    "labels": ["l"],
+    "entities": ["e"],
+    "icon": ["n", "i"],
+    "status": ["n", "s"],
+    "primary-language": ["s", "p"],
+    "secondary-languages": ["s", "s"],
+    "field": ["f"],
+    "field-values": ["fg"],
+    "metadata": ["m"],
+}
+
+
+def translate_label_filters(filters: List[str]) -> List[str]:
+    """
+    Translate friendly filter names to the shortened filter names.
+    """
+    output = []
+    for fltr in filters:
+        if fltr[0] != "/":
+            raise InvalidQueryError(
+                "filters", f"Invalid label filter. It must start with a `/`: {fltr}"
```

Comment:
Since you are reusing the `translate_label_filters` for different params (`faceted` and `filters`), you can't hardcode the param value for the exception here.